lib/dbg/debug_lvl.go

```diff
@@ -272,3 +271,40 @@ func LLvlf2(f string, args ...interface{}) { Lvlf(-1, f, args...) }
 func LLvlf3(f string, args ...interface{}) { Lvlf(-1, f, args...) }
 func LLvlf4(f string, args ...interface{}) { Lvlf(-1, f, args...) }
 func LLvlf5(f string, args ...interface{}) { Lvlf(-1, f, args...) }
+
+// SetDebugVisible set the global debug output level in a go-rountine-safe way
+func SetDebugVisible(lvl int) {
+	debugMut.Lock()
+	defer debugMut.Unlock()
+	debugVisible = lvl
+}
+
+func DebugVisible() int {
```

Comment:
Sorry but since we started on this go-lint / travis stuff, we need at least a one-line comment on each of the following functions.. ! (DebugVisible, SetShowTime,DebugTime, SetUseColors, UseColors)